crest/Assets/Crest/Crest/Scripts/Spline/Spline.cs

```diff
@@ -29,6 +29,16 @@ public partial class Spline : MonoBehaviour, ISplinePointCustomDataSetup
         int _version = 0;
 #pragma warning restore 414
 
+        public enum Offset
+        {
+            Left,
+            Center,
+            Right
+        }
+        [Tooltip("Where generated ribbon should lie relative to spline. If set to Center, ribbon is centered around spline.")]
+        // 01-Oct-21: Default to right for now to support legacy data. In the future we should update this default to Center.
+        public Offset _offset = Offset.Right;
```

Comment:
We could make this a 0-1 range? I don't know if the added flexibility it will give will be useful.

Reply:
i spent some time considering if it should be a (-1, 1) value. i thought maybe it would confuse some people, and also wasnt sure it was needed. im still not 100% sure but this is where i landed. what do you think?

Reply:
perhaps givign less flexibility and then potentially migrating to more flexibility in the future is preferable than giving too much and then deciding to take it away later

Reply:
Make sense in regards to it is later better to give than to take. Let's go with what you have.